CHANGELOG.md

```diff
@@ -1,8 +1,7 @@
 # Changelog
 
 ## Unreleased
-...
-
+- Add `DataClient` and `StudioDatasetRepository` as connectors to Studio for submitting data.
```

Comment:
this should go below ###features in the changelog. Also we could label it as [beta] here